armi/reactor/composites.py

```diff
@@ -617,11 +617,11 @@ def getParameterCollection(cls):
         Return a new instance of the specific ParameterCollection type associated with this object.
 
         This has the same effect as ``obj.paramCollectionType()``. Getting a new
-        instance through a class method like this is useful in situations where the
+        instance through a method like this is useful in situations where the
```

Comment:
This is one of the instances where I think "class method" is the proper term. This is maybe the most significant one, because the whole point of this paragraph is to talk about why the `@classmethod` decorator is useful. This is probably the one case where removing the word "class" changes the meaning of the docstring.

Reply:
Oh, it actually does use `@classmethod`. My bad.  I didn't even check. I didn't think ARMI had too many of those. I'll fix it.